Android/Java/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.java

```diff
@@ -119,51 +134,54 @@ public void showRewardedVideoButtonTapped(View view) {
 
     public void loadInterstitialButtonTapped(View view) {
         // This will load an Interstitial ad
-
+        mInterstitialAd = new LevelPlayInterstitialAd(INTERSTITIAL_AD_UNIT_ID);
+        mInterstitialAd.setListener(new DemoInterstitialAdListener(this));
+        mInterstitialAd.loadAd();
         log("loadInterstitial");
-        IronSource.loadInterstitial();
     }
 
     public void showInterstitialButtonTapped(View view) {
         // It is advised to make sure there is available ad before attempting to show an ad
-        if (IronSource.isInterstitialReady()) {
+        if (mInterstitialAd.isAdReady()) {
             // This will present the Interstitial.
             // Unlike Rewarded Videos there are no placements.
 
             log("showInterstitial");
-            IronSource.showInterstitial();
+            mInterstitialAd.showAd(this);
         } else {
             // load a new ad before calling show
         }
     }
 
     public void loadBannerButtonTapped(View view) {
-        // call IronSource.destroyBanner() before loading a new banner
-        if (bannerParentLayout != null) {
-            this.destroyBanner();
+        // call destroy() before loading a new banner
+        if (mBannerAd != null) {
+            destroyBanner();
         }
 
         // choose banner size
-        // you can pick any banner size: ISBannerSize.BANNER, ISBannerSize.LARGE, ISBannerSize.RECTANGLE, ISBannerSize.SMART or even define a custom banner size by providing width and height
-        ISBannerSize size = ISBannerSize.BANNER;
+        // 1. recommended - Adaptive ad size that adjusts to the screen width
+        // 2. Adaptive ad size using fixed width ad size
+        // 3. Specific banner size - BANNER, LARGE, MEDIUM_RECTANGLE
+        LevelPlayAdSize adSize = LevelPlayAdSize.createAdaptiveAdSize(this);
 
-        // initialize IronSourceBanner object, using the IronSource.createBanner API
-        ironSourceBannerLayout = IronSource.createBanner(this, size);
+//        LevelPlayAdSize adSize = LevelPlayAdSize.createAdaptiveAdSize(this, 400);
+//        LevelPlayAdSize adSize = LevelPlayAdSize.BANNER;
```

Comment:
I would change it little bit to make it clearer:
// choose banner size
// the supported banner sizes are:  BANNER, LARGE, MEDIUM_RECTANGLE, ADAPTIVE.
// we recommend using the adaptive ad size that adjusts to the screen width
`LevelPlayAdSize adSize = LevelPlayAdSize.createAdaptiveAdSize(this);`

// for the fixed sizes (BANNER, LARGE, MEDIUM_RECTANGLE)
// `LevelPlayAdSize adSize = LevelPlayAdSize.Banner` / / example for banner ad size

WDYT?

Reply:
what you wrote is not aligned to the knowledge center, I would recomoend aligning the text to there. see the link below https://developers.is.com/ironsource-mobile/android/banner-integration-sdk-8-1-0-alpha-with-adaptive/#step-3